src/ClientHandle.cpp

```diff
@@ -2320,6 +2320,15 @@ void cClientHandle::SendHealth(void)
 
 
 
+void cClientHandle::SendHideTitle(void)
```

Comment:
Maybe remove this method and if SendSetSubtitle() is called with a empty string, hide the title.